setup.py

```diff
@@ -285,13 +285,16 @@ def add_dir_name(dir_name, filenames):
     else:
         extra_compile_args.extend(min_extra_compile_args)
 
-    # Added -Wall to get all warnings and -Werror to treat them as errors
     extra_compile_args.append("-Wall")
```

Comment:
I would put this line after line 292. Otherwise we obtain a lot of warnings on Windows (at least on my VM)